apis/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
The removal of this is making the format check fail

Reply:
Done. Thank you